tdp_server/services/deployment_plan.py

```diff
@@ -87,6 +87,7 @@ async def from_reconfigure_request(
         latest_successes = db.execute(
             get_latest_success_service_component_version_query()
         ).all()
+        db.close()
```

Comment:
You should not close the `db` here, the `db` variable is given by the caller, you don't know if the caller still needs to query the `db`. Only the function which open the session should close it.